indexing-service/src/main/java/io/druid/indexing/overlord/setup/EqualDistributionWorkerSelectStrategy.java

```diff
@@ -47,7 +47,13 @@ public int compare(
               ImmutableZkWorker zkWorker, ImmutableZkWorker zkWorker2
           )
           {
-            return -Ints.compare(zkWorker2.getCurrCapacityUsed(), zkWorker.getCurrCapacityUsed());
+            int comparation = -Ints.compare(zkWorker2.getCurrCapacityUsed(), zkWorker.getCurrCapacityUsed());
+
+            if(comparation == 0) {
+              comparation = zkWorker2.getWorker().getIp().compareTo(zkWorker.getWorker().getIp());
```

Comment:
`FillCapacityWorkerSelectStrategy` compares workers with capacity(asc) + host base. Two comparators could be merged into one.